test/suite/test_compact07.py

```diff
@@ -175,7 +181,24 @@ def test_compact07(self):
 
         # Check that foreground compaction has done some work on the small table.
         self.assertGreater(self.get_pages_rewritten(uri_small), 0)
+        
+        # Get how many files are currently in the background compaction tracking list.
+        files_tracked = self.get_bg_compaction_files_tracked()
 
+        # Drop the tables and wait for sometime for them to be removed from the background 
+        # compaction server list.
+        for i in range(self.n_tables):
+            uri = self.uri_prefix + f'_{i}'
+            self.dropUntilSuccess(self.session, uri)
+        
+        self.session.checkpoint()
+        
+        # The tables should get removed from the tracking list once they exceed the max idle time 
+        # after they're dropped. The background compact debug setting sets the max idle time to 30 
+        # seconds.
+        while self.get_bg_compaction_files_tracked() >= files_tracked:
```

Comment:
As discussed, this should be updated with a comment saying we are dropping all the tables but 1. Only 1 should be tracked.